### PR TITLE
Don't generate doc if doc comments only filled with 'white' characters

### LINKED ISSUE #39339: Doc: 'empty' documentation block causes display issues
Using `///` blocks with whitespaces only displaces the associated declaration. This behaviour isn't displayed when no whitespaces are used.

It's fairly easy to reproduce:
```rust
// will cause display issues, note the whitespaces
///    
pub fn foo() {}

// ok, no whitespaces
///
pub fn bar() {}
```

cc @GuillaumeGomez

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #39339.

r? @steveklabnik 

cc @rust-lang/docs

### CODE REVIEW

File: src/librustdoc/html/markdown.rs [hunk 1]
```diff
@@ -237,8 +237,9 @@ pub fn render(w: &mut fmt::Formatter,
             let my_opaque: &MyOpaque = &*((*opaque).opaque as *const MyOpaque);
             let text = (*orig_text).as_bytes();
             let origtext = str::from_utf8(text).unwrap();
+            let origtext = origtext.trim_left();
             debug!("docblock: ==============\n{:?}\n=======", text);
-            let rendered = if lang.is_null() {
+            let rendered = if lang.is_null() || origtext.len() == 0 {
```
Comment (frewsxcv) [inline src/librustdoc/html/markdown.rs]: Nit: `origtex.is_empty()`

**Review by frewsxcv — APPROVED** [review]

File: src/librustdoc/html/markdown.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/librustdoc/html/markdown.rs]: Good point!

**Comment by GuillaumeGomez** [comment]
Updated.

**Pull request merged**